docs/core_docs/docs/integrations/tools/goat.mdx

```diff
@@ -0,0 +1,99 @@
+---
+hide_table_of_contents: true
+---
+
+# GOAT
+
+[GOAT](https://github.com/goat-sdk/goat) is the finance toolkit for AI agents.
+
```

Comment:
Please run `yarn format` from root and add the following warning:

```
:::warning
This tool exists outside of the main LangChain repository [here](https://github.com/goat-sdk/goat/).
Please use caution when linking wallets to external providers and make sure they are trusted.
:::
```

Reply:
Ran yarn format, tho it didn't change anything it seems? Prob missing something

Reply:
Don't forget the warning!

Reply:
Done!